src/main/java/slimeknights/tconstruct/tools/data/ModifierRecipeProvider.java

```diff
@@ -88,6 +93,33 @@ private void addItemRecipes(Consumer<IFinishedRecipe> consumer) {
                           .addIngredient(TinkerTags.Items.WITHER_BONES)
                           .addCriterion("has_bone", hasItem(TinkerTags.Items.WITHER_BONES))
                           .build(withCondition(consumer, ConfigEnabledCondition.WITHER_BONE_CONVERSION), location(folder + "wither_bone_conversion"));
+    
+    // TODO: use addCastingWithCastRecipe
+    FluidStack debris = new FluidStack(TinkerFluids.moltenDebris.get(), MaterialValues.INGOT);
```

Comment:
These should probably melt back into debris as well. Can make one recipe that accepts all 3 as input probably